components/Navigation.js

```diff
@@ -2,30 +2,29 @@ import styled from "styled-components";
 import Image from "next/image";
 import homeIcon from "../public/images/homeIcon.png";
 import addIcon from "../public/images/addIcon.png";
-import { useRouter } from "next/router";
+import Link from "next/link";
 
 export default function Navigation() {
-  const router = useRouter();
   return (
     <StyledNavigation>
       <UlStyledList>
         <StyledList>
-          <SyledHomeButton onClick={() => router.push("/")} type="button">
+          <Link href="/">
             <Image src={homeIcon} alt="Go to Homepage" />
-          </SyledHomeButton>
+          </Link>
```

Comment:
u don't need styling for the Links?
Otherwise you could write
```javascript
<Link href="/" passHref>
  <NavLink aria-label="Go to Homepage">
    <Image src={homeIcon} alt="Go to Homepage" />
  </NavLink>
</Link>

const NavLink = styled.a`
  width: 3em;
  height: 3em;
  border: 0em;
  background-color: transparent;

  &:hover {
    cursor: pointer;
    filter: invert(1);
  }
`;
```